src/web/core/src/main/java/org/geoserver/web/data/layer/CascadedWFSStoredQueryAbstractPage.html

```diff
@@ -18,7 +18,7 @@
 
 <wicket:fragment wicket:id="newPage">
   <div>
-    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName"></input>
+    <wicket:message key="chooseNativeName">Layer name </wicket:message> <input type="text" wicket:id="nativeName">
```

Comment:
From what I read <input> is valid in HTML, if you want valid XHTML is should be <input/> instead.
I'm not sure, but believe that some of our CSS was depending on the browser not to enter in quircks mode, and wondering if using <input> alone would do that (sorry, little web frontend experience...)